lib/lighthouse/benefits_intake/sidekiq/submission_status_job.rb

```diff
@@ -0,0 +1,191 @@
+
+    def log(level, msg, **payload)
+      this = self.class.name
+      message = format('%<class>s: %<msg>s', { class: this, msg: msg.to_s })
+      Rails.logger.public_send(level, message, class: this, **payload)
+    end
+
+    def batch_process(pending_attempts)
+      intake_service = BenefitsIntake::Service.new
+
+      pending_attempts.each_slice(batch_size) do |batch|
+        batch_uuids = batch.map(&:benefits_intake_uuid)
+        log(:info, 'processing batch', batch_uuids:)
+
+        response = intake_service.bulk_status(uuids: batch_uuids)
+
+        log(:info, 'bulk status response', response:)
+        raise response.body unless response.success?
+
+        next unless (data = response.body['data'])
+
+        handle_response(data)
+      end
+    rescue => e
+      log(:error, 'ERROR processing batch', message: e.message)
+    end
+
+    def pending_attempts_hash
+      @pah ||= FormSubmissionAttempt.where(aasm_state: 'pending').includes(:form_submission)
+                                    .index_by(&:benefits_intake_uuid)
+    end
+
+    # @see https://developer.va.gov/explore/api/benefits-intake/docs
+    def handle_response(response_data)
+      response_data.each do |submission|
+        uuid = submission['id']
+
+        next unless pending_attempts_hash[uuid]
+
+        # Log the status for debugging
+        status = submission.dig('attributes', 'status')
+        log(:info, "Processing submission UUID: #{uuid}, Status: #{status}")
+
+        update_attempt_record(uuid, status, submission)
+        monitor_attempt_status(uuid, status)
+
+        handle_attempt_result(uuid, status)
+      end
+    end
+
+    def update_attempt_record(uuid, status, submission)
+      form_submission_attempt = pending_attempts_hash[uuid]
+      lighthouse_updated_at = submission.dig('attributes', 'updated_at')
+
+      case status
+      when 'expired'
+        # Indicates that documents were not successfully uploaded within the 15-minute window.
+        error_message = 'expired'
+        form_submission_attempt.fail!
+
+      when 'error'
+        # Indicates that there was an error. Refer to the error code and detail for further information.
+        error_message = "#{submission.dig('attributes', 'code')}: #{submission.dig('attributes', 'detail')}"
+        form_submission_attempt.fail!
+
+      when 'vbms'
+        # Submission was successfully uploaded into a Veteran's eFolder within VBMS
+        form_submission_attempt.vbms!
+      end
+
+      form_submission_attempt.update(lighthouse_updated_at:, error_message:)
+    end
+
+    def monitor_attempt_status(uuid, status)
+      form_submission_attempt, result = attempt_status_result(uuid, status)
+      form_id = form_submission_attempt.form_submission.form_type
+
+      metric = "#{STATS_KEY}.#{form_id}.#{result}"
+      StatsD.increment(metric)
+      StatsD.increment("#{STATS_KEY}.all_forms.#{result}")
+
+      level = result == 'failure' ? :error : :info
+      payload = {
```

Comment:
Again here. Doesn't the new track_request handle this bit too?

Reply:
it does but would have the payload down a level (under 'additional_context') which will mess with any monitors people have setup